### ISSUE TITLE
Manifest within multiple collections

### ISSUE THREAD

**Issue by mikeapp** [ProjectMirador/mirador-design#5]
The design could show the Manifest as a member of more than one Collection.

**Comment by aeschylus** [comment]
Can you expand on this. Would this be essentially a back-link of some kind rendered from the `within` field, if it is available?

**Comment by mikeapp** [comment]
Yes, `within` referencing more than one Collection.   I wasn't 100% sure what the mock was attempting to show there, but assumed it was referencing IIIF Collection(s).

**Comment by ggeisler** [comment]
Good point, @mikeapp. Yes, the mockup is referencing IIIF collections. We'll revise the bottom of the Window Panel design to accommodate this possibility.

**Comment by azaroth42** [comment]
Where in the mocks are Collections implemented? If an album is a Manifest and an "item" a Canvas, then you need a structure above an "album".
